client/core/trade.go

```diff
@@ -1739,11 +1717,11 @@ func (c *Core) sendRedeemAsync(t *trackedTrade, match *matchTracker, coinID, sec
 		auth := &match.MetaData.Proof.Auth
 		auth.RedeemSig = ack.Sig
 		auth.RedeemStamp = encode.UnixMilliU(time.Now())
-		if match.Match.Side == order.Maker {
-			// As maker, this is the end. However, this diverges from server,
-			// which still needs taker's redeem.
-			match.SetStatus(order.MatchComplete)
```

Comment:
This `SetStatus` was already set as taker, hence the condition.  But it's OK to have a redundant SetStatus to be safe.  Was there another reason to remove the condition?  The implications of setting a status are that the status is changing, so if debugging this it's going to look like a bug for taker, at least without a comment.